Add CAP200DG and CAP300DG as ALIAS

evanshultz opened the pull request:
Also mild cleanup to original CapZero family descriptions and keywords

https://ac-dc.power.com/sites/default/files/product-docs/capzero_family_datasheet.pdf
https://ac-dc.power.com/sites/default/files/product-docs/capzero-2_family_datasheet.pdf
https://ac-dc.power.com/sites/default/files/product_document/data_sheet/capzero-3_family_datasheet.pdf

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

cpresser commented:
- [ ] Some of the aliases still have a link to http://www.powerint.com/sites/default/files/product-docs/capzero_family_datasheet.pdf which is down.
- [ ] The voltage does not seem to match. 825V vs. 875V
- [ ] Cosmetic (IMHO not blocking the merge): The units in description/keywords do not match. 3.5uF in description, 3500nF in keywords.

evanshultz commented:
Thanks for looking!
1. Updated to new URL.
1. Maybe the datasheet was changed since the description was originally 875V but the current datasheet is 825V. Or just a typo. Oh well. Fixed.
1. Removed the redundant voltage and capacitance keywords.
